<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>627</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>0Arame</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(36, 38, 39);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>0Arame</family>
      <pointsize>32</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(80, 82, 83);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>0Arame</family>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(223, 223, 223);</string>
    </property>
    <property name="text">
     <string>RPM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fanSpeedSendButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>0Arame</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
border-width: 4px;
border-color: white;</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QSlider" name="percentSlider">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal {
    height: 3px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: rgba(255,255,255,50);
    margin: 2px 0;
    border-radius: 1px;
}

QSlider::sub-page:horizontal {
    margin: 1px 0; /*handle is placed by default on the contents rect of the groove. Expand outside the groove */
	background: white;
	border-radius: 2px;
}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="currentPercent">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>0Arame</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
